wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,72 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+Users are getting banned for several reasons not limited to:
+-   Continuous misbehaviour regardless of how many infringements were given.
```

Comment:
you can't really "give" an infringement, so swap this for "silences", or "how many infringements they accumulate"